app/pages/prisma.tsx

```diff
@@ -0,0 +1,12 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+export async function getAllQuestions() {
+    try {
+        const questions = await prisma.question.findMany();
+        return questions;
+    } catch(e) {
+        console.log("error message: ", e);
+    }
+}
```

Comment:
I think we should discuss where to handle data fetching in our app, I don't think it should belong in this directory